src/sync/db.rs

```diff
@@ -219,28 +219,6 @@ impl Database {
         ))
     }
 
-    /// Creates a new collection with encrypted fields, automatically creating new data encryption
```

Comment:
There's now no sync api for this, but that's consistent with the rest of `ClientEncryption` - we made the decision a while back to defer any work on that until/unless there are user requests for it.